pkg/controller/routes.go

```diff
@@ -134,6 +135,9 @@ func routes(router *router.Router, cfg *rest.Config, registryTransport http.Roun
 	router.Type(&appsv1.Deployment{}).Namespace(system.ImagesNamespace).HandlerFunc(networkpolicy.ForBuilder)
 	router.Type(&netv1.NetworkPolicy{}).Selector(managedSelector).HandlerFunc(gc.GCOrphans)
 
+	router.Type(&internaladminv1.ImageRoleAuthorizationInstance{}).HandlerFunc(permissions.BumpImageRoleAuthorizations)
+	router.Type(&internaladminv1.ClusterImageRoleAuthorizationInstance{}).HandlerFunc(permissions.BumpClusterImageRoleAuthorizations)
```

Comment:
Will those only get triggered upon create/update/delete?

Reply:
The handler will get triggered when an IRA is created/updated/deleted or when the controller starts up. However, if the IRA hasn't changed (the observed generation is the same as the generation) or the app doesn't have any denied permissions, then we won't do anything.